<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>624</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Алгоритм Дейкстры перевода в обратную польскую запись</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>780</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="clean_show"/>
    <widget class="QWidget" name="main_menu">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>461</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="menu">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="Theory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Теория</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Demo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Демонстрация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Train">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Тренировка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Change">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Поменять пользователя</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="demo" name="demo_show"/>
    <widget class="train" name="training_show"/>
    <widget class="theory" name="theory_show"/>
    <widget class="admin" name="admin_show"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="Information">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>119</y>
      <width>242</width>
      <height>108</height>
     </rect>
    </property>
    <property name="title">
     <string>О программе...</string>
    </property>
    <addaction name="prog_info"/>
    <addaction name="Statistica"/>
   </widget>
   <addaction name="Information"/>
  </widget>
  <action name="prog_info">
   <property name="text">
    <string>Информация</string>
   </property>
  </action>
  <action name="Statistica">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>theory</class>
   <extends>QWidget</extends>
   <header location="global">theory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>admin</class>
   <extends>QWidget</extends>
   <header location="global">admin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>train</class>
   <extends>QWidget</extends>
   <header location="global">train.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>demo</class>
   <extends>QWidget</extends>
   <header location="global">demo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
